Khan/khan-exercises
# Add option for EU style right angle notation

**ISSUE**: https://khanacademy.atlassian.net/browse/IC-591

**Summary:**
Some countries do not use US-style right angle notation, instead they
use an arc with a dot inside, see
https://en.wikipedia.org/wiki/Right_angle#/media/File:Triangle_30-60-90_rotated.png
Here we add new options to `drawPolygon()` and `drawAngle()` so that translators are able to localize such graphies easily.

The code is inspired by a custom Graphie code made by Charlotte Auen for translators, see:
https://international-forum.khanacademy.org/t/right-angle-notation-in-graphie-images/400

As a follow-up, these changes should be ported to interactive widgets (which live in webapp) so that they can be autolocalized.

**Test plan:**
 - Pull these changes in your local `graphie-to-png` repo
 - in graphie-to-png run `python app.py` and open port 5001
 - Load template 13
 - Line61: add property `dottedRightAngleMarkers: true`
 - Regraph, rectangle inside the triangle should change
   into an arc with a dot
 - Line 121: Change angle from 120 to 90
 - Line 129: Add property `dottedRightAngleMarker: true`
 - Regraph, you should see 90° angle in a circle, arc with a dot

Here's how it should look:
![image](https://user-images.githubusercontent.com/9539441/74478726-36ddc100-4eae-11ea-8941-de3fb5733e41.png)
